go.mod

```diff
@@ -1,6 +1,8 @@
 module go.viam.com/rdk
 
-go 1.23
+go 1.23.0
+
+toolchain go1.23.1
```

Comment:
I'm having some trouble understanding why my `go mod tidy` inserted [this](https://go.dev/ref/mod#go-mod-file-toolchain) and what it does? cc @abe-winter

Reply:
Thanks; tried to remove.